.jshintignore

```diff
@@ -0,0 +1,33 @@
+# Logs
```

Comment:
Is there a point to this file? Snitch doesn't need it, and all of our other libs don't have one either. If Snitch did need it, it'd make sense, but I think it can go. If you do see a use case, share.